tests/cross/src/test/scala/tests/pc/PcRenameSuite.scala

```diff
@@ -488,4 +488,28 @@ class PcRenameSuite extends BasePcRenameSuite {
        |  } yield b
        |""".stripMargin,
   )
+
+  check(
+    "end-marker".tag(IgnoreScala2),
+    """|def <<he@@llo>>(a: Int) =
+       |  ???
+       |end <<hello>>
+       |""".stripMargin,
+  )
+
+  check(
+    "end-marker-with-comment".tag(IgnoreScala2),
+    """|def <<he@@llo>>(a: Int) =
+       |  ???
+       |end /* a comment */ <<hello>> /* a comment */
+       |""".stripMargin,
+  )
+
+  check(
+    "end-marker-wrong",
```

Comment:
```suggestion
    "end-marker-wrong".tag(IgnoreScala2),
```
Missing test tag? :slightly_smiling_face: